### PR TITLE
Add Total Books To Book Log Form

### PR DESCRIPTION
**Pull request by maxsebso**
## This PR

- Added a running count at the bottom of the Book Log popup of the total amount of books that are currently in the log

## Screenshots

Screen Shot 2021-08-17 at 12 58 38 PM

### CODE REVIEW

**Comment by rymaju** [comment]
Not a huge fan of the styling. I would remove the border stroke and make the font size consistent with the rest of the text

File: src/components/schoolDirectory/BookLogsMenu.tsx [hunk 1]
```diff
@@ -87,6 +106,27 @@ const BookLogsMenu: React.FC<BookLogsMenuProps> = ({
...
+          notes: log.notes,
+          style: ind > added - 1 ? '#fff' : '#E6F7FF',
+        };
+        return styledLog;
+      });
+  };
+
+  const calculateTotalBooks = (data: BookLogWithStyling[]) => {
+    let total = 0;
+    data.map((elem) => (total += elem.count));
```
Comment (rymaju) [inline src/components/schoolDirectory/BookLogsMenu.tsx:126]: This shouldnt be a map, it should be a reduce (foldr) hello fundies 1
```
const total = data.reduce((a,b)=>a+b,0)
```

**Review by rymaju — COMMENTED** [review]
I would want to fix the summing function, but also maybe the styling too

**Comment by granttebeau** [comment]
Agreed with Ryan's comments above^
Styling should be improved and you can remove the calculateTotalBooks function and make it into a reduce. But once those changes are made, looks good to me

**Comment by maxsebso** [comment]
Screen Shot 2021-09-02 at 11 27 43 AM

new style

**Review by rymaju — APPROVED** [review]
image

⛏️ Margin of text should be aligned, IMO I also think we should reduce the font size to be the same as the rest of the text on this page.

**Pull request merged**